Security/Util/String.php

```diff
@@ -5,6 +5,8 @@
 /**
  * String utility functions.
  *
+ * @deprecated This class will break in PHP 7; use hash_equals() instead
```

Comment:
I couldn't find any reference to this class anywhere in this project! Just the test.